src/printer.ts

```diff
@@ -413,8 +420,8 @@ export class PugPrinter {
 			this.currentLineLength += 1;
 			let tempToken: AttributeToken | EndAttributesToken = this.nextToken;
 			let tempIndex: number = this.currentIndex + 1;
-			let nonPrefixAttributes: number = 0;
 			let hasPrefixAttribute: boolean = false;
+			let hasNonPrefixAttributes: boolean = false;
```

Comment:
I even didn't know myself what `hasPrefixAttribute` represents :sweat_smile: 
But isn't `hasNonPrefixAttributes` the opposite of `hasPrefixAttribute`
Maybe some inline comments could help here

Reply:
Well, they can both be true, so the answer I think is _no_.  Think of them as `hasSomePrefixAttribute`, `hasSomeNonePrefixAttributes`. An element can have both types of attributes.

As far as I understand it, a prefix attribute is an id or class attribute that is inside the list of attributes and should move outside, to the element (e.g. `div(id="foo", class="bar")` becoming `#foo.bar`. And a non-prefix attribute is anything that stays inside the parentheses after formatting.

 I think the naming is actually fine as it is now? Or would you like to suggest better naming for these?

Reply:
So this is the only remaining g open question here I think

Reply:
I can add a comment that explains the above?

Reply:
Yes, this would be nice

Reply:
I think I found a better way to do this now, see my latest commit. With this, the PR is finished from my side.